src/utils.ts

```diff
@@ -1,59 +1,36 @@
-import { ResolutionType } from './typings';
+import { ImageListType } from './typings';
+
+export const triggerOpenDialog = (inputRef): void => {
```

Comment:
How about this name?
```suggestion
export const openFileDialog = (inputRef): void => {
```